extensions/vertx-http/runtime/src/main/java/io/quarkus/devui/runtime/config/AuthJsonRPCService.java

```diff
@@ -0,0 +1,124 @@
+package io.quarkus.devui.runtime.config;
+
+import static io.quarkus.devui.runtime.config.PermissionSet.QUARKUS_NULL_VALUE;
+
+import java.util.*;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.inject.Inject;
+
+import org.jboss.logging.Logger;
+
+import io.quarkus.dev.console.DevConsoleManager;
+import io.quarkus.devui.runtime.comms.JsonRpcMessage;
+import io.quarkus.devui.runtime.comms.MessageType;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+@ApplicationScoped
+public class AuthJsonRPCService {
+    private static final Logger LOG = Logger.getLogger(AuthJsonRPCService.class.getName());
+    private final List<String> METHODS = List.of("GET", "POST", "PUT", "DELETE", "PATCH", "OPTIONS", "HEAD", "CONNECT",
+            "TRACE"); // do not change order
+    private final List<String> AUTH_MECHANISM = List.of(QUARKUS_NULL_VALUE, "basic", "bearer", "form"); // do not change order
+    private final List<String> POLICIES = List.of("permit", "deny", "authenticated"); // do not change order
```

Comment:
Note `// do not change order` without explanation is not as useful as if you added bit more. btw fact that it is order sensitive seems strange.

Reply:
I concur; I'll enhance the comment for clarity. It's worth noting that the sequence is non-critical and remains functional in any order. The arrangement mirrors the frontend presentation, specifically within the context of Vaadin components such as vaadin-list-box and vaadin-select.